src/main/java/FrameworksDrivers/ScheduleGraphics.java

```diff
@@ -46,31 +46,24 @@
     JLabel sundayLabel;
 
     // addEventButton popup
-    // TODO: keep declaration hear, move creation inside
+    // TODO: keep declaration here, move creation inside
     JTextField addEventName = new JTextField();
     JComboBox addEventDate = new JComboBox(new String[]{"Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"});
-    String[] timesList = {"1:00am", "2:00am", "3:00am", "4:00am", "5:00am", "6:00am", "7:00am", "8:00am", "9:00am", "10:00am", "11:00am", "12:00pm", "1:00pm", "2:00pm", "3:00pm", "4:00pm", "5:00pm", "6:00pm", "7:00pm", "8:00pm", "9:00pm", "10:00pm", "11:00pm", "12:00am"};
+    String[] timesList = {"12:00am", "1:00am", "2:00am", "3:00am", "4:00am", "5:00am", "6:00am", "7:00am", "8:00am", "9:00am", "10:00am", "11:00am", "12:00pm", "1:00pm", "2:00pm", "3:00pm", "4:00pm", "5:00pm", "6:00pm", "7:00pm", "8:00pm", "9:00pm", "10:00pm", "11:00pm"};
```

Comment:
Right now we have our inputs from users as a 24-clock system, should we keep the displays the same? Or keep it like this and just make sure to equate each one when we integrate the other parts?

Reply:
yes, a part later in the code converts the strings into ints, but they still need to be modified to match the 24 hour clock system